src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String DATE_FORMAT = "dd.MM.yyyy";
+    private static final String HEADING = "Report for period %s - %s";
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern(DATE_FORMAT);
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder info = new StringBuilder(String.format(HEADING, dateFrom, dateTo));
+        LocalDate firstLimit = LocalDate.parse(dateFrom, FORMATTER);
```

Comment:
As for me it is better rename variables to something like `localDateFrom` / `localDateTo`  or `startDate` /`endDate` .